moveit_core/collision_detection/src/collision_tools.cpp

```diff
@@ -45,21 +45,21 @@ void getCostMarkers(visualization_msgs::msg::MarkerArray& arr, const std::string
                     std::set<CostSource>& cost_sources)
 {
   std_msgs::msg::ColorRGBA color;
-  color.r = 1.0f;
-  color.g = 0.5f;
-  color.b = 0.0f;
-  color.a = 0.4f;
+  color.r{ 1.0f };
+  color.g{ 0.5f };
+  color.b{ 0.0f };
+  color.a{ 0.4f };
```

Comment:
This sort of initialization does not work on struct members like this. See the build error it produces in CI.